### DISCUSSION TITLE
Why Topic not support UTF8 ?

### DISCUSSION THREAD

**Discussion by mitchellhu** [hazzelnuts/ntfy-for-delphi#43] [General]
Try Vcl sample with traditional Chinese, Japanese, and Korean Topics.
It will raise an Error:  Project VCLPublisher.exe raised exception class EIdHTTPPrototocolExcepton with message 'HTTP/1.1 400 Bad Request'
So,  Is it a UTF8-supporting issue on Sever side or the API restriction?

**Comment by mitchellhu** [comment]
Sorry, For my stupid, it is should be a Unicode issue, not UTF8.

**Comment by p-samuel** [comment]
Hi @mitchellhu, could you provide me some example of this? I think'd be better open a issue is this case.

**Reply by mitchellhu** [reply]
Try these ,please
Traditional Chinese Topic: 中文主題
Japanese Topic: 日本のテーマ
Korean Topic: 한국어 테마
